### ISSUE TITLE
[Accessibility][Name,Role,value - About]: Ensures buttons have discernible text.

### ISSUE THREAD

**Issue by ShellySoni** [microsoft/BotFramework-Composer#2125]
[edit by @corinagum] 
The issues filed below have been addressed, but there are a couple other related problems that I would like addressed in this issue. To quote my comment [below](https://github.com/microsoft/BotFramework-Composer/issues/2125#issuecomment-613142023): 

> [R]elated to this PR, I believe the about page needs to be modified to remove `tabIndex={-1}`, and the `<IconBotton>` should be changed to just an icon inside `<Link>`

Once the above is addressed, we can return this to 'waiting for verification'

---

**User Experience:**
This issue should be resolved for good coding practice if buttons have discernible text.

**Test Environment:**
Environment: Chromium Edge (Anaheim) + Narrator
Build Version: 2004 (OS Build 19564.1005)​​
Microsoft Edge Version: 82.0.425.0 (Official build) canary (64-bit)​
Test URL: http://localhost:3000/home

**Repro Steps:**
1. Open URL: http://localhost:3000/home
2. "Bot Framework Composer" homepage will gets opened.
3. Select any of the previously created bot from "Recent Bots" or create one from the available Bots under 'Examples' and activate it.
4. Select 'About' from left navigation.
5. Run K4W tool on the page and Observe the issue

**Actual Result:**
'Terms of Use' and 'Privacy' buttons does not have discernible text. Screen reader read only as button.

This accessibility issue was found using Accessibility Insights for Web(keros), a tool that helps debug and find accessibility issues earlier. Get more information & download this tool at http://aka.ms/Keros.

**Expected Result:**
Ensures buttons i.e. 'Terms of Use' and 'Privacy' have discernible text. Screen reader should read the name of the button.

**Note:**
Same issue observed for 'Design Flow' pane.
Same issue observe throughout the website for every property pane. {Attached some of screenshot below, please look the issue throughout the application}

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Attachment:**
[MAS4.1.2_About_K4W.zip](https://github.com/microsoft/BotFramework-Composer/files/4266576/MAS4.1.2_About_K4W.zip)
![ensures button have discernible text](https://user-images.githubusercontent.com/60919065/75652681-b70f6f00-5c81-11ea-8169-128f2da3846b.png)

**Comment by corinagum** [comment]
![MAS4 1 2_About_K4W](https://user-images.githubusercontent.com/14900841/75575129-67783b80-5a14-11ea-810c-6f5305dc510c.png)

.png uploaded for ease of access; note that the .zip still has a video file.

**Comment by beyackle** [comment]
This looks like it's not happening any more.

![image](https://user-images.githubusercontent.com/61990921/77360646-f8d48800-6d0a-11ea-8cdf-3bed43927576.png)

**Comment by ashish315** [comment]
@beyackle As checked on below environment the issue is still repro.
1. Same issue observed for 'Design Flow' pane.
2. Same issue observe throughout the website for every property pane. {Attached some of screenshot below, please look the issue throughout the application}

**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
Browser: Microsoft Edge Canary {Version 83.0.474.0 (Official build) canary (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

Attachments-
![image](https://user-images.githubusercontent.com/60913131/78450583-4a7efa00-769d-11ea-9b00-919406684b2c.png)
![image](https://user-images.githubusercontent.com/60913131/78450775-a39b5d80-769e-11ea-8d7c-0148b59e9726.png)

**Comment by corinagum** [comment]
@beyackle related to this PR, I believe the about page needs to be modified to remove `tabIndex={-1}`, and the `<IconBotton>` should be changed to just an icon inside `<Link>`
I didn't see another a11y issues addressing these problems so would you mind if we put this back in To Do and use this issue to further address the About page?

**Comment by beyackle** [comment]
Sure - go ahead.

**Comment by corinagum** [comment]
Updated the top comment. Thank you.

**Comment by ashish315** [comment]
As checked, the issue is resolved now. So, HCL is closing the issue.
